apps/app/src/app/shared/util/services/settings.service.ts

```diff
@@ -169,10 +155,10 @@ export class SettingsService {
    *
    * @returns Observable<string>
```

Comment:
Il faut chancher ça !
```suggestion
   * @returns Observable
```